examples/grid_example/public/app.tsx

```diff
@@ -314,7 +277,7 @@ export const GridExample = ({
             useCustomDragHandle={true}
             renderPanelContents={renderPanelContents}
             onLayoutChange={onLayoutChange}
-            css={customLayoutCss}
+            css={layoutStyles}
```

Comment:
I can't see where this is getting called - in Emotion can you send a function into `css`? And how does it end up getting the correct `euiTheme`? I must be missing something because it works correctly in light + dark mode.

Reply:
Yup!! This is the "syntactic sugar" described [here](https://github.com/elastic/eui/discussions/6828#discussioncomment-11247425) - I personally think it's super clean 🤌

Reply:
I've found that this error is thrown regardless of using the `useEuiTheme` hook or not - it seems to happen for all Emotion in my experience. Did you see something different @mbondyra?